bucket/flutter.json

```diff
@@ -27,11 +30,7 @@
         "Write-Host 'Some licenses need to be accepted before developing. It is recommended to do by running ''flutter doctor --android-licenses''.' -ForegroundColor Yellow",
         "flutter doctor"
     ],
-    "bin": [
-        "bin\\flutter.bat",
-        "flutter-dev-setup.ps1"
-    ],
-    "env_add_path": "bin\\cache\\dart-sdk\\bin",
+    "env_add_path": ["bin","bin\\cache\\dart-sdk\\bin"],
```

Comment:
@zoritle can you separate the two PRs?

Replacing depends with suggest would make it good to go very quickly but I think we would like more discussion when it comes to env_add_path and bin removals